homeassistant/components/vicare/sensor.py

```diff
@@ -815,6 +815,38 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         state_class=SensorStateClass.MEASUREMENT,
         value_getter=lambda api: api.getHumidity(),
     ),
+    ViCareSensorEntityDescription(
+        key="ventilation_demand",
+        translation_key="ventilation_demand",
+        value_getter=lambda api: _filter_states(api.getVentilationDemand().lower()),
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        device_class=SensorDeviceClass.ENUM,
+        options=["ventilation"],
+    ),
```

Comment:
Would it be viable to only add this when we know more states? Because I don't think this is that useful now

Reply:
```suggestion
```